pkg/export/setup/setup.go

```diff
@@ -117,7 +123,7 @@ func FromFlags(a *kingpin.Application, userAgent string) func(log.Logger, promet
 		Default(opts.ProjectID).StringVar(&opts.ProjectID)
 
 	a.Flag("export.user-agent", "Override for the user agent used for requests against the GCM API.").
```

Comment:
Please at least update the comment saying that this variable won't override the actual user agent.

Suggest saying that the actual user-agent is still subjected by the underlying libraries, but this variable provides hints for the final user-agent format.